core/src/main/java/org/apache/iceberg/ManifestEntry.java

```diff
@@ -103,27 +103,6 @@ default boolean isLive() {
    */
   void setDataSequenceNumber(long dataSequenceNumber);
 
-  /**
-   * Returns the data sequence number of the file if the entry status is ADDED or EXISTING.
-   * Otherwise, returns the sequence number of the snapshot in which the file was removed.
-   *
-   * <p>Note that usage of this method should be avoided as it behaves inconsistently for different
-   * entry statutes. Use {@link #dataSequenceNumber()} instead.
-   *
-   * @deprecated since 1.0.0, will be removed in 1.1.0; use {@link #dataSequenceNumber()} instead.
-   */
-  @Deprecated
-  Long sequenceNumber();
```

Comment:
@aokolnychyi could you please review all places that adjust sequenceNumber -> dataSequenceNumber?